src/model/entities/test/test-entity-factory.ts

```diff
@@ -0,0 +1,52 @@
+import { Rundown, RundownInterface } from '../rundown'
+import { Segment, SegmentInterface } from '../segment'
+import { Part, PartInterface } from '../part'
+import { Piece, PieceInterface } from '../piece'
+
+export class TestEntityFactory {
+  public static createRundown(rundownInterface?: Partial<RundownInterface>): Rundown {
+    if (!rundownInterface) {
+      rundownInterface = {} as RundownInterface
+    }
```

Comment:
You can get around this by changing the parameter to having a default value:
```typescript
  public static createRundown(rundownInterface: Partial<RundownInterface> = {}): Rundown {
```

Reply:
Updated